examples/PromptInjectionDetector/PromptInjectionGPT4o.ipynb

```diff
@@ -0,0 +1,439 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "vigvBmpNxHvb"
+   },
+   "source": [
+    "# BrainTrust Classification Tutorial (Prompt Injection)\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/braintrustdata/braintrust-cookbook/blob/main/examples/PromptInjectionDetector/PromptInjectionGPT4o.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>\n",
+    "\n",
+    "This is a quick tutorial on how to build and evaluate an AI app to classify prompt injection attempts with [BrainTrust](https://www.braintrust.dev/).\n",
+    "\n",
+    "Before starting, make sure that you have a BrainTrust account. If you do not, please [sign up](https://www.braintrust.dev) first. After this tutorial, learn more by visiting [the docs](http://www.braintrust.dev/docs).\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "c0hvsPRZLCUz"
+   },
+   "source": [
+    "First, we'll install some dependencies.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "rTIA4DVQw1K7"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": []
+    }
+   ],
+   "source": [
+    "%pip install -U braintrust openai datasets autoevals python-dotenv --quiet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll load our environmental variables which has OpenAI and Brain Trust keys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "load_dotenv() "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll import the [prompt-injections dataset](https://huggingface.co/datasets/deepset/prompt-injections) from Huggingface.\n"
```

Comment:
Can you add a sentence or two of context about the dataset? Maybe it's worth adding a bit about what prompt injection is above? Not a ton, but I feel like one or two sentences will really help, eg "Prompt injection is when ...." and then "This dataset contains XYZ (number) input and output pairs that ...."